tests/e2e/specs/gtag-events/blocks-pages.test.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * External dependencies
+ */
+const { test, expect } = require( '@playwright/test' );
+
+/**
+ * Internal dependencies
+ */
+import {
+	createSimpleProduct,
+	setSettings,
+	clearSettings,
+} from '../../utils/api';
+import {
+	blockProductAddToCart,
+	relatedProductAddToCart,
+	simpleProductAddToCart,
+} from '../../utils/customer';
+import {
+	createProductCollectionBlockShopPage,
+	createProductsBlockShopPage,
+} from '../../utils/create-page';
+import { getEventData, trackGtagEvent } from '../../utils/track-event';
+
+const config = require( '../../config/default' );
+const simpleProductPrice = parseFloat( config.products.simple.regular_price );
+
+let simpleProductID;
+
+test.describe( 'GTag events on block pages', () => {
+	test.beforeAll( async () => {
+		await setSettings();
+		simpleProductID = await createSimpleProduct();
```

Comment:
:nail_care: 
As we use `simpleProductID` only in the scope of `GTag events on block pages` and do not clear it after, I'd move the declaration there. To avoid artifacts when we add another `describe`.
```suggestion
test.describe( 'GTag events on block pages', () => {
	let simpleProductID;
	
	test.beforeAll( async () => {
		await setSettings();
		simpleProductID = await createSimpleProduct();
```